crates/aptos-ledger/src/lib.rs

```diff
@@ -143,17 +144,96 @@ pub fn get_app_name() -> Result<String, AptosLedgerError> {
     }
 }
 
-/// Returns the public key of your Aptos account in Ledger device
+/// Returns the next available account in Ledger starting from start_index to end_index, inclusive
```

Comment:
Are we sure it is inclusive? I think when you iterate through a `Range` it is inclusive on the lower bound and exclusive on the upper bound: https://doc.rust-lang.org/std/ops/struct.Range.html.